src/common/transformations/include/ov_ops/rotary_positional_embeddings.hpp

```diff
@@ -0,0 +1,38 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "openvino/op/op.hpp"
+#include "transformations_visibility.hpp"
+
+namespace ov {
+namespace op {
+namespace internal {
+
+///
+/// \brief Rotary Positional Embeddings operation
+/// Internal operation which may change in the future
+/// \ingroup ov_ops_cpp_api
+class TRANSFORMATIONS_API RPE : public ov::op::Op {
```

Comment:
RoPE is more common abbreviation for this operation.

Reply:
Discussed the question offline.